stubs/Authlib/authlib/oauth2/rfc6749/grants/base.pyi

```diff
@@ -0,0 +1,56 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Collection
+from typing import Any
+from typing_extensions import TypeAlias
+
+from authlib.oauth2 import OAuth2Request
+from authlib.oauth2.rfc6749 import ClientMixin
+
+_SERVER_RESPONSE: TypeAlias = tuple[int, str, list[tuple[str, str]]]
+
+class BaseGrant:
+    TOKEN_ENDPOINT_AUTH_METHODS: Collection[str]
+    GRANT_TYPE: str | None
+    TOKEN_RESPONSE_HEADER = Collection[tuple[str, str]]  # noqa: Y026
```

Comment:
I think this shouldn't be a type alias, since it's initialized to this value: https://github.com/lepture/authlib/blob/master/authlib/consts.py#L7

`TOKEN_RESPONSE_HEADER: Collection[tuple[str, str]] = ...`


Relevant test failure:
```
error: authlib.oauth2.rfc6749.grants.base.BaseGrant.TOKEN_RESPONSE_HEADER is inconsistent, runtime is not a type
Stub: in file stubs/Authlib/authlib/oauth2/rfc6749/grants/base.pyi:14
Type alias for typing.Collection[tuple[builtins.str, builtins.str]]
Runtime:
[('Content-Type', 'application/json'), ('Cache-Control', 'no-store'), ('Pragma', 'no-cache')]
```